pages.de/common/gcc.md

```diff
@@ -0,0 +1,24 @@
+# gcc
+
+> Präprozessiert und kompiliert C und C++ Quellcodedateien und linkt diese anschließend zusammen.
+> Mehr Informationen: <https://gcc.gnu.org>.
+
+- Kompiliert mehrere Quellcodedateien zu einer ausführbaren Datei:
+
+`gcc {{Quelldatei1.c}} {{Quelldatei2.c}} -o {{Ausgabebinärdatei}}`
```

Comment:
We use `snake_case` for variable names, `Ausgabebinärdatei` requires start with mayus? the same for `Pfad`, etc.

Reply:
nouns in german start with uppercase